sourcecoop/egg-source-coop.json

```diff
@@ -0,0 +1,152 @@
set.\r\nGITHUB_PACKAGE=\"ampreeT\/SourceCoop\"\r\n\r\nif [ -z \"${GITHUB_USER}\" ] && [ -z \"${GITHUB_OAUTH_TOKEN}\" ] ; then\r\n    echo -e \"using anon api call\"\r\nelse\r\n    echo -e \"user and oauth token set\"\r\n    alias curl='curl -u ${GITHUB_USER}:${GITHUB_OAUTH_TOKEN} '\r\nfi\r\n\r\n# get release info and download links\r\nLATEST_JSON=$(curl --silent \"https:\/\/api.github.com\/repos\/${GITHUB_PACKAGE}\/releases\/latest\")\r\nRELEASES=$(curl --silent \"https:\/\/api.github.com\/repos\/${GITHUB_PACKAGE}\/releases\")\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_URL=$(echo ${LATEST_JSON} | jq .assets | jq -r .[].browser_download_url | grep -i ${MATCH})\r\nelse\r\n    VERSION_CHECK=$(echo ${RELEASES} | jq -r --arg VERSION \"${VERSION}\" '.[] | select(.tag_name==$VERSION) | .tag_name')\r\n    if [ \"${VERSION}\" == \"${VERSION_CHECK}\" ]; then\r\n        DOWNLOAD_URL=$(echo ${RELEASES} | jq -r --arg VERSION \"${VERSION}\" '.[] | select(.tag_name==$VERSION) | .assets[].browser_download_url' | grep -i ${MATCH})\r\n    else\r\n        echo -e \"defaulting to latest release\"\r\n        DOWNLOAD_URL=$(echo ${LATEST_JSON} | jq .assets | jq -r .[].browser_download_url)\r\n    fi\r\nfi\r\ncurl -L ${DOWNLOAD_URL} -o sourcecoop.zip\r\nunzip sourcecoop.zip -d $HOME\/bms\r\n\r\n## Create custom server.cfg\r\necho -e 'Creating custom \"server.cfg\" configuration file...'\r\ncd $HOME\/bms\/cfg\r\nrm -f server.cfg\r\ncat > server.cfg << EOF\r\n\/\/ Black Mesa server.cfg file\r\nhostname \"Black Mesa: Coop\"\r\nsv_password 0\r\n\r\n\/\/ provides lag-free noblock, shows teammates as friendly\r\n\/\/ and puts an arrow above their head (srccoop handles team switching)\r\nmp_teamplay 1\r\n\r\n\/\/ set 1 if you hate your friends\r\nmp_friendlyfire 0\r\n\r\n\/\/ setting 1 disables sprinting\r\nsv_always_run 0\r\n\r\n\/\/ disables map timelimit\r\nmp_timelimit 0\r\n\r\n\/\/ changes maps in empty server\r\nmp_mapcycle_empty_timeout_seconds 3600\r\n\r\n\/\/ Uncomment to loop back to the first map after finishing a campaign\r\n\/\/ Leave commented to use nextmap (mapcycle) instead\r\nsourcecoop_homemap \"bm_c0a0a\"\r\n\r\n\/\/ Enables pre-caching of map data (eg. models, etc.)\r\n\/\/ Disabling this may resolve crashing\/hanging on certain maps (namely bm_c4a4a), but may cause performance issues on other maps.\r\nmod_forcetouchdata 1\r\n\r\nrcon_password \"\" \/\/ disable rcon\r\nsv_tags \"custom, coop, sourcecoop\"\r\nsv_cheats 0\r\nsv_alltalk 1\r\nEOF\r\n\r\n## Check for successful installation.\r\ncd $HOME\/bms\/addons\r\nif [ -d \"metamod\" ] && [ -d \"sourcemod\" ] && [ -f \"sourcemod\/plugins\/srccoop.smx\" ]; then\r\n    echo -e \"\\nSourceCoop Dedicated Server successfully installed!\\n\"\r\nelse\r\n    echo -e \"\\n\\n[Error] One or more SourceCoop dependancies failed to install!\"\r\n    echo -e \"\\tTry checking your [Advanced] startup settings and\/or reinstalling the server again.\\n\"\r\n    exit 1\r\nfi",
```

Comment:
Please use at least the full steamcmd install line if ever branches or logins want to be done that then their is only the need for just extra variables

Reply:
That's fair; I'll add this in tomorrow as well :)

Reply:
Done. Double check my syntax though if you don't mind 👍